scripts/archive_everything

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3.7
+
+"""Tell the comet-archiver to archive everything currently in redis."""
+
+import aioredis
+import asyncio
+import json
+import logging
+
+from comet import broker
+
+REDIS_LIST_STATES = "states"
+REDIS_LIST_DATASETS = "datasets"
+REDIS_TODO_STATES = "state"
+REDIS_TODO_DATASETS = "dataset"
+
+
+async def send_to_archiver(redis, list_name, archive_list_name):
+    """
+    Send a state or dataset to comet-archiver.
+
+    Parameters
+    ----------
+    redis : aioredis.ConnectionPool
+        a redis instance
+    list_name : str
+        Must match the redis lists that store stuff, atm only "states" or "datasets"
+    archive_list_name : str
+        Must match the redis lists that store archive-to do's, atm only "state" or "dataset"
```

Comment:
We mourn at having these concepts distinguished only by an `s`.
We thank Rick for storing them in very distinctly named constant variables.

Reply:
:man_facepalming: